docs/theme-styling.md

```diff
@@ -0,0 +1,43 @@
+# Theme Styling
+
+We're using a block theme in WordPress. The styling can be customized through `Appearance > Theme > Customize` in WP Admin dashboard. You can then access the "Styles" menu item to check further on theme styling.
+
+Read more about [block themes in WordPress](https://www.wpzoom.com/blog/what-are-wordpress-block-themes/#:~:text=Block%20themes%20simplify%20the%20website,customization%20directly%20from%20the%20editor.).
+
+## Custom CSS
+
+### Pre-requisites
+- Install Dart Sass - https://sass-lang.com/install
+- Verify sass installation
+    ```sh
+    sass --version
+    ```
+
+**NOTE:** Sass comes on multiple platforms and language. Ensure to install Dart Sass.
+
+### Details
+Custom styling is handled within the `goonj-crm` theme directory. The CSS is structured as per the following:
+```
+- goonj-crm/
+    - style.scss
+    - style.css
+    - assets/
+        - styles/
+            - variables.scss
+            - main.scss
+            - md.scss
+            - lg.scss
+            - xl.scss
+```
+
+Run the following command to compile the changes and generate the stylesheet:
+```sh
+cd path/to/theme
+sass style.scss style.css --style compressed
```

Comment:
Can we also not generate a source map – especially for production builds (although currently we have only one build)?

```suggestion
sass style.scss style.css --style compressed --no-source-map
```

Reply:
Some drawbacks of including sourcemaps that I've came across:

1. **Increased Load Time**: Source maps increase the size of the files that need to be downloaded, which can slow down the initial load time of your application.
1. **Unnecessary Information**: Source maps are not needed for the application to run, so including them in production adds unnecessary files and complexity to your deployment.
1. **Security Risks**: Source maps can expose your original source code, including comments and unminified code, making it easier for malicious users to find vulnerabilities or understand your code logic.